ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -244,7 +239,7 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withUrlParams(), withCd
           </h3>
           <div style={styles.infoBox} data-test-id='cdrVersion'>
             <div style={styles.infoBoxHeader}>Dataset</div>
-            <div style={{fontSize: '0.5rem'}}>{cdrVersion ? cdrVersion.name : 'Loading...'}</div>
+            <div style={{fontSize: '0.5rem'}}>{workspace ? cdrVersion(workspace, cdrVersionListResponse).name : 'Loading...'}</div>
```

Comment:
is checking for the workspace sufficient here? What if cdrVersionListResponse is still loading?

Reply:
There's a comment on the `withCdrVersions()` HOC: "Rendering of the connected component is blocked on initial load of the CDR versions."

Reply:
Oops.  Looks like you were right: #4191